lib/pdk/cli/exec.rb

```diff
@@ -135,7 +135,8 @@ def run_process!
           begin
             @process.start
           rescue ChildProcess::LaunchError => e
-            raise PDK::CLI::FatalError, _("Failed to execute '%{command}': %{message}") % { command: argv.join(' '), message: e.message }
+            raise PDK::CLI::FatalError, _('Failed to execute process: %{message}') % { message: e.message } unless @process.respond_to?(:argv)
+            raise PDK::CLI::FatalError, _("Failed to execute '%{command}': %{message}") % { command: @process.argv.join(' '), message: e.message }
```

Comment:
First raise is conditional. Rubocop preferred this _raise x unless y_ form. I'll see if I can re-arrange it so that it is easily readable and passes rubocop.